### DISCUSSION TITLE
ChaoticSoliloquy-4x8B model crashes

### DISCUSSION THREAD

**Discussion by waynekwal** [YellowRoseCx/koboldcpp-rocm#44] [Q&A]
I'm trying out this MoE build: https://huggingface.co/xxx777xxxASD/ChaoticSoliloquy-4x8B

When using Koboldcpp-rocm, no matter what settings I use, it crashes with an access issue (0x000000001), I think, I don't get a log on crash).  I've tried different quants and quant builders, full offload to VRAM or no offload, and also Vulcan as well.  Crashes before UI load every time.  I've let the builder know as well, but I thought I should post it here. 

[EDIT] I should also mention I normally unckeck mmq per the release notes, but I have tried it both ways just in case.

**Comment by Dajinu** [comment]
This model just doesn't work with the current koboldcpp-rocm release. I'm waiting for an update too.

**Reply by waynekwal** [reply]
Gotcha.  Is this a delay from regular Kobold or something about the model?  Do we know?

**Reply by Dajinu** [reply]
I have no idea. Haven't seen any reaction from the devs yet.
